03-casestudies.Rmd

```diff
@@ -375,7 +375,7 @@ Using data from NYC Open Data, this interactive visualization shows the variety
 
 We can see what some of the familiar and uncommon trees planted in the five boroughs of New York City are. This visualization allows one to see the distribution quickly. One can make inferences based on the distribution, such as trees in the Bronx and Manhattan seem to be distributed more uniformly compared to the other three boroughs. It gives a direct comparison between the five boroughs which could be used to make a compelling decision by the audience.
 
-![NYC Street Trees](images/NY_trees.png)
+![NYC Street Trees](https://github.com/jymhe120/datasciencecoursera/blob/master/NY_trees.png)
```

Comment:
I think the link for NY_trees.png should be from the reader, as in, you just update the file in the images folder and the link remains as 'images/NY_trees.png'
Don't upload it to your GitHub account and then link it.